apisix/core/request.lua

```diff
@@ -269,6 +269,15 @@ function _M.get_port(ctx)
 end
 
 
+function _M.get_path(ctx, original)
```

Comment:
The `original` is unused?

Reply:
Yes, a useless code that I forgot to change.

Reply:
fixed